rtfscluster/rtfs_cluster.go

```diff
@@ -55,8 +55,9 @@ func (cm *ClusterManager) GenClient() error {
 func (cm *ClusterManager) ParseLocalStatusAllAndSync() ([]gocid.Cid, error) {
 	// this will hold all the cids that have been synced
 	var syncedCids []gocid.Cid
-	// only fetch the local status for all pins
-	pinInfo, err := cm.Client.StatusAll(true)
+	// only fetch the local status for all pins - todo: double check if this is
+	// the right TrackerStatusError to user
+	pinInfo, err := cm.Client.StatusAll(api.TrackerStatusError, true)
```

Comment:
@postables weird dependency updates from vendor rebuild, this is one - `api.TrackerStatusError` is just a guess, not sure if right?

Reply:
Ill leave it for now, we can remove in another PR

Reply:
Sounds good